spring-cloud-task-docs/src/main/asciidoc/features.adoc

```diff
@@ -33,7 +33,7 @@ Spring Boot application configured to be a task (annotated with the `@EnableTask
 annotation).
 
 At the beginning of a task (before any `CommandLineRunner` or `ApplicationRunner`
-implementations have been executed, an entry in the `TaskRepository` is created recording
+implementations) have been executed, an entry in the `TaskRepository` is created recording
```

Comment:
Nice catch but probably will remove the opening parenthesis instead.  I'll fix this on merge.